wal/doc.go

```diff
@@ -38,8 +38,9 @@ Each WAL file is a stream of WAL records. A WAL record is a length field and a w
 protobuf. The record protobuf contains a CRC, a type, and a data payload. The length field is a
 64-bit packed structure holding the length of the remaining logical record data in its lower
 56 bits and its physical padding in the first three bits of the most significant byte. Each
-record is 8-byte aligned so that the length field is never torn. The CRC contains the CRC32
-value of all record protobufs preceding the current record.
+record is 8-byte aligned, which is the multiple times of the minimum sector size 512, so that the
```

Comment:
"record 8-byte aligned, which divides the minimum sector size of 512 bytes, so that the length field is never torn."?

Reply:
@heyitsanthony 
Sorry to bother you. I'm a little bit confused about 8-byte alignment, implemented in [encodeFrameSize](https://github.com/coreos/etcd/blob/master/wal/encoder.go#L99-L107) function. 
Here is my thought: The way to avoid torn-wirte is writing data atomic to disk. Since the minimum operation unit of disk is sector, writing data atomic means a related sector is either written completely or not been written at all. Why could 8-byte alignment achieve this target?
Thanks in advance.

Reply:
the field is guaranteed to never straddle a sector and it's never partially written out; assuming sector writes are all-or-nothing, the field is either entirely written on fsync or not written at all